### ISSUE TITLE
haveSameElementsAs prints the iterables in wrong order

### ISSUE THREAD

**Issue by GoogleCodeExporter** [etorreborre/specs#80]
```
List(1,2) must haveSameElementsAs(List(3,4))

produces

'[3, 4]' doesn't have the same elements as '[1, 2]'

It should produce

'[1, 2]' doesn't have the same elements as '[3, 4]'

For example the equalTo matcher prints the compared objects in that order

specs version 1.5.0
```

Original issue reported on code.google.com by `saynomoo@gmail.com` on 17 Jun 2009 at 8:47

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by r1057.
```

Original comment by `etorrebo...@gmail.com` on 22 Jun 2009 at 1:45
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
The fix is available in 1.5.1-SNAPSHOT
```

Original comment by `etorrebo...@gmail.com` on 27 Jun 2009 at 11:39

**Comment by GoogleCodeExporter** [comment]
Original comment by `etorrebo...@gmail.com` on 14 Feb 2010 at 2:43
- Changed state: **Verified**
